src/java/org/apache/cassandra/tools/nodetool/SetAutoRepairConfig.java

```diff
@@ -53,7 +53,7 @@ public class SetAutoRepairConfig extends AbstractCommand
                                                           "|allow_parallel_replica_repair|allow_parallel_repair_across_schedules" +
                                                           "|materialized_view_repair_enabled|repair_max_retries" +
                                                           "|repair_retry_backoff|repair_session_timeout|min_repair_task_duration" +
-                                                          "|repair_by_keyspace|token_range_splitter.<property>]" })
+                                                          "|repair_by_keyspace|token_range_splitter.<property>]|mixed_major_version_repair_enabled" })
```

Comment:
```suggestion
                                                          "|repair_by_keyspace|mixed_major_version_repair_enabled|token_range_splitter.<property>]" })
```

Suggest moving it before `token_range_splitter.<property>` since that's a bit of an exceptional thing.  Really we should order this alphabetically, but I think its fine as is for now.